### PR TITLE
Added KopiaUI YouTube tutorial links

### PR DESCRIPTION
**Pull request by jkowalski**

### CODE REVIEW

File: site/content/docs/Getting started/_index.md [hunk 1]
```diff
@@ -11,7 +11,18 @@ Make sure to familiarize yourself with Kopia [features](../features/) before fol
...
 ## Installation
 
-Kopia is distributed as a single binary called `kopia` (except on Windows, where it's called `kopia.exe`). To install it follow the [Installation Guide](../installation/).
+Kopia is distributed as a single command-line (CLI) binary called `kopia`. 
+
+To install it follow the [Installation Guide](../installation/).
+
+## Kopia UI
+
+If you prefer graphical user interface,
```
Comment (ntolia) [inline site/content/docs/Getting started/_index.md]:
```suggestion
If you prefer a graphical user interface,
```

File: site/content/docs/Getting started/_index.md [hunk 2]
```diff
@@ -11,7 +11,18 @@ Make sure to familiarize yourself with Kopia [features](../features/) before fol
...
+Kopia comes with a user friendly desktop app for Windows, macOS and Linux called `KopiaUI`which runs in the background and allows you to create snapshots, define policies and restore files quickly. More advanced features require the use of CLI tool.
```
Comment (ntolia) [inline site/content/docs/Getting started/_index.md]:
```suggestion
Kopia comes with a user-friendly desktop app for Windows, macOS, and Linux called `KopiaUI`which runs in the background and allows you to create snapshots, define policies and restore files quickly. More advanced features require the use of the CLI tool.
```

File: site/content/docs/Getting started/_index.md [hunk 3]
```diff
@@ -11,7 +11,18 @@ Make sure to familiarize yourself with Kopia [features](../features/) before fol
...
+
+Kopia UI is new and experimental. See the tutorial on Youtube:
```
Comment (ntolia) [inline site/content/docs/Getting started/_index.md]:
```suggestion
The Kopia UI is new and experimental. See the tutorial on Youtube:
```

File: site/content/docs/Installation/_index.md [hunk 4]
```diff
@@ -4,7 +4,15 @@ linkTitle: "Installation"
 weight: 2
 ---
 
-The easiest way to get started with Kopia is to download pre-compiled Kopia binary. Alternatively you compile it yourself from source. 
+Kopia supports both command line (CLI) and graphical (GUI) user interfaces.
+
+### Command Line Tool
+
+The easiest way to get started with Kopia CLI tool is to download pre-compiled `kopia` binary. Alternatively you compile it yourself from source.
```
Comment (ntolia) [inline site/content/docs/Installation/_index.md]:
```suggestion
The easiest way to get started with the Kopia CLI tool is to download pre-compiled `kopia` binary. Alternatively, you can compile it yourself from source. 
```

File: site/content/docs/Installation/_index.md [hunk 5]
```diff
@@ -4,7 +4,15 @@ linkTitle: "Installation"
...
-The easiest way to get started with Kopia is to download pre-compiled Kopia binary. Alternatively you compile it yourself from source. 
+Kopia supports both command line (CLI) and graphical (GUI) user interfaces.
+
+### Command Line Tool
+
+The easiest way to get started with Kopia CLI tool is to download pre-compiled `kopia` binary. Alternatively you compile it yourself from source. 
+
+### Kopia UI
+
+You can download installer packages of KopiaUI for Windows, macOS and Linux from [Releases](https://github.com/kopia/kopia/releases/latest) page on GitHub.
```
Comment (ntolia) [inline site/content/docs/Installation/_index.md]:
```suggestion
You can download installer packages of KopiaUI for Windows, macOS, and Linux from the [Releases](https://github.com/kopia/kopia/releases/latest) page on GitHub.
```

File: site/content/docs/Installation/_index.md [hunk 6]
```diff
@@ -16,6 +24,7 @@ Kopia is released as a single, stand-alone binary, available for many operating
 
 To download latest binary for your platform, go to the [Releases](https://github.com/kopia/kopia/releases/latest) page on GitHub.
 
+
```
Comment (ntolia) [inline site/content/docs/Installation/_index.md]:
```suggestion
```
spurious change?

File: site/content/docs/Getting started/_index.md [hunk 7]
```diff
@@ -11,7 +11,18 @@ Make sure to familiarize yourself with Kopia [features](../features/) before fol
...
+
+[![Kopia UI Tutorial](https://img.youtube.com/vi/sHJjSpasWIo/0.jpg)](https://www.youtube.com/watch?v=sHJjSpasWIo)
```
Comment (ntolia) [inline site/content/docs/Getting started/_index.md]: This doesn't do the right thing. Can you try the Hugo shortcode for Youtube? https://gohugo.io/content-management/shortcodes/#example-youtube-input

Comment (jkowalski) [reply site/content/docs/Getting started/_index.md]: done, thanks for the suggestion

File: site/content/docs/Getting started/_index.md [hunk 8]
```diff
@@ -11,7 +11,18 @@ Make sure to familiarize yourself with Kopia [features](../features/) before fol
...
 
-Kopia is distributed as a single binary called `kopia` (except on Windows, where it's called `kopia.exe`). To install it follow the [Installation Guide](../installation/).
+Kopia is distributed as a single command-line (CLI) binary called `kopia`. 
+
+To install it follow the [Installation Guide](../installation/).
+
+## Kopia UI
+
+If you prefer a graphical user interface,
+Kopia comes with a user-friendly desktop app for Windows, macOS, and Linux called `KopiaUI`which runs in the background and allows you to create snapshots, define policies and restore files quickly. More advanced features require the use of the CLI tool.
```
Comment (julio-lopez) [inline site/content/docs/Getting started/_index.md]:
```suggestion
Kopia comes with a user-friendly desktop app for Windows, macOS, and Linux called `KopiaUI`, which runs in the background and allows you to create snapshots, define policies and restore files quickly. More advanced features require the use of the CLI tool.
```

**Pull request merged**